cmd/mapper/mapper.go

```diff
@@ -86,33 +86,47 @@ func main() {
 					"datasetid: %s, "+
 					"accessionids: %v, "+
 					"error: %v)",
-					d.CorrelationId,
+					delivered.CorrelationId,
 					mappings.DatasetID,
 					mappings.AccessionIDs,
 					err)
+
+				// Nack message so the server gets notified that something is wrong and requeue the message
+				if e := delivered.Nack(false, true); e != nil {
+					log.Errorf("Failed to nack following getheader error message "+
```

Comment:
The error message is not very clear. Maybe sth like:
```suggestion
					log.Errorf("Failed to nack message (failed to map files to dataset)"+
```
?